bin/lambda-cfn.js

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env node
+
+var cli = require('../lib/cli');
+var d3 = require('d3-queue');
+
+var parsed;
+try { parsed = cli.parse(process.argv.slice(2), process.env); }
+catch (err) { return finished(err); }
+
+// if (parsed.options.all) {
+  // call a helper function to compile the list of
+  // stacks for the command to loop over
+  //var q = d3.queue(1);
+  //for (var stack in parsed.options.stacks) {
+    // modify parsed and pass to command loop
+    // q.defer(cli.main, modified_parsed)
+    // q.await(function(err,data) {});
+//  }
+// } else {
+  cli.main(parsed, finished);
+
+  function finished(err, data) {
+    if (err) process.stderr.write(err.message + '\n');
```

Comment:
@zmully - is there a benefit or special reason to use `process.stderr.write()` and `process.stdout.write()` over `console.error()` and `console.log()` here? Per https://stackoverflow.com/a/33534019 it looks like `console.log` is a wrapper for `process.stdout.write()` that automatically adds a new line and accepts any data type. It also allows for `%` associations.